changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
```

Comment:
```suggestion
```

Let's cut this table. It returns data in the stange/hard-to-use data format. We can make it better later.